app/main.py

```diff
@@ -1,3 +1,26 @@
+from app.Knight import Knight
+
+
+def fight(knight1: Knight, knight2: Knight) -> None:
+    knight1.hp -= knight2.power - knight1.protection
+    knight2.hp -= knight1.power - knight2.protection
+    if knight1.hp <= 0:
+        knight1.hp = 0
+    if knight2.hp <= 0:
+        knight2.hp = 0
```

Comment:
You can make method in class Knight for this check.

You have similar recommendation in thje checklist:
![image](https://user-images.githubusercontent.com/104218550/202272130-de74eb61-9e3e-4285-ad50-ea9b8ea75167.png)

Reply:
Done